tools/spec-gen-sdk/src/automation/entrypoint.ts

```diff
@@ -1,74 +1,18 @@
-import {existsSync, mkdirSync, readFileSync, rmSync} from 'fs';
+import { existsSync, mkdirSync, rmSync } from 'fs';
 import * as winston from 'winston';
-import { getRepoKey, RepoKey } from '../utils/repo';
-import {
-  FailureType,
-  setFailureType,
-  WorkflowContext,
-  workflowInit,
-  workflowMain
-} from './workflow';
-import {
-  loggerConsoleTransport,
-  loggerDevOpsTransport,
-  loggerFileTransport,
-  loggerTestTransport,
-  loggerWaitToFinish,
-  sdkAutoLogLevels
-} from './logging';
+import { workflowInit, workflowMain } from './workflow';
+import { loggerConsoleTransport, loggerDevOpsTransport, loggerFileTransport, loggerTestTransport, loggerWaitToFinish, sdkAutoLogLevels, vsoLogError } from './logging';
 import path from 'path';
 import { generateReport, generateHtmlFromFilteredLog, saveFilteredLog, saveVsoLog } from './reportStatus';
-import { SpecConfig, SdkRepoConfig, getSpecConfig, specConfigPath } from '../types/SpecConfig';
-import { getSwaggerToSdkConfig, SwaggerToSdkConfig } from '../types/SwaggerToSdkConfig';
+import { getSpecConfig, specConfigPath } from '../types/SpecConfig';
+import { getSwaggerToSdkConfig } from '../types/SwaggerToSdkConfig';
 import { extractPathFromSpecConfig } from '../utils/utils';
-import { toolError } from '../utils/messageUtils';
-
-interface SdkAutoOptions {
-  specRepo: RepoKey;
-  sdkName: string;
-  branchPrefix: string;
-  localSpecRepoPath: string;
-  localSdkRepoPath: string;
-  tspConfigPath?: string;
-  readmePath?: string;
-  pullNumber?: string;
-  apiVersion?: string;
-  runMode: string;
-  sdkReleaseType: string;
-  specCommitSha: string;
-  specRepoHttpsUrl: string;
-  workingFolder: string;
-  headRepoHttpsUrl?: string;
-  headBranch?: string;
-  runEnv: 'local' | 'azureDevOps' | 'test';
-  version: string;
-  skipSdkGenFromOpenapi?: string;
-}
-
-export type SdkAutoContext = {
-  config: SdkAutoOptions;
-  logger: winston.Logger;
-  fullLogFileName: string;
-  filteredLogFileName: string;
-  htmlLogFileName: string;
-  vsoLogFileName: string;
-  specRepoConfig: SpecConfig;
-  sdkRepoConfig: SdkRepoConfig;
-  swaggerToSdkConfig: SwaggerToSdkConfig
-  isPrivateSpecRepo: boolean;
-};
-
-/*
- * VsoLogs is a map of task names to log entries. Each log entry contains an array of errors and warnings.
- */
-export type VsoLogs = Map<string, {
-  errors?: string[];
-  warnings?: string[];
-}>;
+import { FailureType, SdkAutoContext, SdkAutoOptions, WorkflowContext } from '../types/Workflow';
+import { getSdkRepoConfig, loadConfigContent, setFailureType } from '../utils/workflowUtils';
 
 export const getSdkAutoContext = async (options: SdkAutoOptions): Promise<SdkAutoContext> => {
   const logger = winston.createLogger({
-    levels: sdkAutoLogLevels.levels
+    levels: sdkAutoLogLevels.levels,
```

Comment:
Can you revert all the style changes? such as tailing comma, tailing colon, etc. We could make that lint and style changes later in one PR.